lib/helpers_.rb

```diff
@@ -1,6 +1,7 @@
 include Nanoc::Helpers::XMLSitemap
 include Nanoc::Helpers::Rendering
 include Nanoc::Helpers::LinkTo
+require 'nokogiri'
```

Comment:
We'll need to add nokogiri to the gemfile so that it gets loaded on bundle install.